tools/launch.ts

```diff
@@ -158,6 +158,11 @@ const parachains: { [name: string]: ParachainConfig } = {
     chain: "moonbase-local",
     docker: "purestake/moonbeam:v0.26.0",
   },
+  "moonbase-0.27.0": {
```

Comment:
The launch conf of a client must be created when the client is released, because we can't yet know which version of the relay will match. This step is listed in the GH release issue, so we won't forget it.

Reply:
Yes, @nbaztec you should add the v0.26.0 here

Reply:
The v0.26.0 was already added, we just have to remove these diff.